heartbeat/README.mariadb-semi-sync.md

```diff
@@ -0,0 +1,149 @@
+Setting up the MariaDB Semi Sync resource agent
+===============================================
+
+Before embarking on this quest one should read the MariaDB pages on replication
+and global transaction IDs, GTID. This will greatly help in understanding what
+is going on and why.
```

Comment:
It would be nice to include a link here. Since the docs tend to be version-specific, we're probably stuck with just linking to the top of the official docs, but that's better than nothing.